From: Baoquan He <bhe@redhat.com>
Subject: mm/swapfile.c: update the code comment above swap_count_continued()
Date: Fri, 7 Feb 2025 10:50:37 +0800

Now, swap_count_continued() has two callers, __swap_duplicate() and
__swap_entry_free_locked(), the relevant code comment is stale.  Update it
to reflect the current situation.

Link: https://lkml.kernel.org/r/Z6V0/UvG1fvkQ4t/@fedora
Signed-off-by: Baoquan He <bhe@redhat.com>
Cc: Chris Li <chrisl@kernel.org>
Cc: Kairui Song <ryncsn@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/swapfile.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/swapfile.c~mm-swapfilec-update-the-code-comment-above-swap_count_continued-v2
+++ a/mm/swapfile.c
@@ -3794,7 +3794,7 @@ outer:
  * when the original swap_map count is decremented from 0 with continuation,
  * borrow from the continuation and report whether it still holds more.
  * Called while __swap_duplicate() or caller of __swap_entry_free_locked()
- * holds swap or cluster lock.
+ * holds cluster lock.
  */
 static bool swap_count_continued(struct swap_info_struct *si,
 				 pgoff_t offset, unsigned char count)
_
